app/models/manager_refresh/inventory/persister.rb

```diff
@@ -0,0 +1,95 @@
+class ManagerRefresh::Inventory::Persister
+  attr_reader :manager, :target, :collections
+
+  # @param manager [ManageIQ::Providers::BaseManager] A manager object
+  # @param target [Object] A refresh Target object
```

Comment:
could you put an OR (`|`) in there? knowing it is an `ActiveRecord::Base` (or similar) seems to be helpful